src/operators/rasterscaling.md

```diff
@@ -0,0 +1,77 @@
+# RasterScaling
+
+The raster scaling operator scales/unscales the values of a raster by a given scale factor and offset.
```

Comment:
can you put each sentence in a separate line

Reply:
i reworked the docs in this commit: [aaa9676](https://github.com/geo-engine/docs/pull/33/commits/aaa967690250857a15aecf0b2993d2172004f14f)